.npmrc

```diff
@@ -1 +1,2 @@
 package-lock=false
+audit=false
```

Comment:
At the risk of being (very) pernickety, there's no new line at the end of this file, which is inconsistent and might cause problems with tooling down the line

Reply:
Will sort it, thanks :)